docs/config/pod_config.md

```
@@ -0,0 +1,159 @@
+<hr style="height:1px;border:none;color:#333;">
+<h1 align="center">Pod Configuration (podConfig)</h1>
+<hr style="height:1px;border:none;color:#333;">
+
+The `podConfig` field is available under both `configuration.velero` and `configuration.nodeAgent` in the DataProtectionApplication (DPA) CR. It allows customizing the pod scheduling and runtime configuration for Velero Deployment pods and Node Agent DaemonSet pods respectively.
+
+All available `podConfig` fields can be discovered from your cluster using:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)